docs/recipes/jspm-systemjs.md

```diff
@@ -27,25 +28,29 @@ You can find more information about setting up Babel with AVA in the [babelrc re
 ### JSPM Loader Helper
 
 You will need to install the [AVA JSPM loader](https://github.com/skorlir/ava-jspm-loader) as a dev dependency.
+Per Babel 7 you will also need to install [@babel/register](https://www.npmjs.com/package/@babel/register).
 
 ```
 $ npm install --save-dev ava-jspm-loader
 ```
 
+```
+npm install @babel/register --save-dev
```

Comment:
Commands should start with `$` and flags should come after `install`.

Reply:
Could you also update other PRs to address this?

Reply:
can do